web/tests/list_widget.test.js

```diff
@@ -756,7 +756,7 @@ run_test("render item", () => {
         name: "replace-list",
         modifier: (item) => `<tr data-item=${item.value}>${item.text}</tr>\n`,
         get_item,
-        html_selector: (item) => `tr[data-item='${item}']`,
+        html_selector: (item) => `tr[data-item='${item.value}']`,
```

Comment:
`html_selector` should return `JQuery`. Same thing for `widget_3` in this test case.